src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
```

Comment:
In the BEM methodology, margin should be avoided for the BEM block elements. Instead, consider using padding in the parent element or creating an additional wrapper element for spacing.